pkgs/development/python-modules/kikit/default.nix

```diff
@@ -0,0 +1,87 @@
+{ bc
+, zip
+, lib
+, fetchFromGitHub
+, bats
+, buildPythonPackage
+, pythonOlder
+, callPackage
+, kicad
+, numpy
+, click
+, markdown2
+, commentjson
+, wxPython_4_2
+, pcbnew-transition
+, pybars3
+, pytest
+, versioneer
+}:
+let
+  properCaseName = "KiKit";
+  solidpython = callPackage ./solidpython { };
+
+  # https://github.com/yaqwsx/KiKit/issues/574
+  # copy-pasted from nixpkgs#8d8e62e74f511160a599471549a98bc9e4f4818d
+  shapely = callPackage ./shapely { };
+in
+buildPythonPackage rec {
+  pname = lib.toLower properCaseName;
+  version = "1.3.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "yaqwsx";
+    repo = properCaseName;
+    rev = "v${version}";
+    hash = "sha256-kDTPk/R3eZtm4DjoUV4tSQzjGQ9k8MKQedX4oUXYzeo=";
+  };
+
+  propagatedBuildInputs =
+    [
+      kicad
+      numpy
+      click
+      markdown2
+      commentjson
+      # https://github.com/yaqwsx/KiKit/issues/575
+      wxPython_4_2
+      pcbnew-transition
+      pybars3
+      shapely
+      # https://github.com/yaqwsx/KiKit/issues/576
+      solidpython
+    ];
+
+  nativeBuildInputs = [
+    versioneer
+    bc
```

Comment:
Why do we need bc?

Reply:
kikit uses `bc` in its Makefile: https://github.com/yaqwsx/KiKit/blob/63301c22ff36b13cc73536b0d289422d3121e772/Makefile#L58

Reply:
Marking this resolved. Please reopen if you have any concerns!